scanner/templates/node/Dockerfile

```
@@ -1,38 +1,53 @@
-FROM debian:bullseye as builder
+# syntax = docker/dockerfile:1
 
-ENV PATH=/usr/local/node/bin:$PATH
 ARG NODE_VERSION={{ .nodeVersion }}
+FROM node:${NODE_VERSION}-slim as base
+
+LABEL fly_launch_runtime="{{ .runtime }}"
+
+ENV NODE_ENV production
+
 {{ if .yarn -}}
 ARG YARN_VERSION={{ .yarnVersion }}
-{{ end }}
-RUN apt-get update; apt install -y curl python-is-python3 pkg-config build-essential && \
-    curl -sL https://github.com/nodenv/node-build/archive/master.tar.gz | tar xz -C /tmp/ && \
-    /tmp/node-build-master/bin/node-build "${NODE_VERSION}" /usr/local/node && \
-{{ if .yarn -}}
-    npm install -g yarn@$YARN_VERSION && \
-{{ end -}}
-    rm -rf /tmp/node-build-master
+RUN npm install -g yarn@$YARN_VERSION
```

We can combine these as well I think

```suggestion
RUN npm install -g yarn@{{ .yarnVersion }}
```

Here as well, I would think having a major version would be better